SofaKernel/framework/sofa/helper/logging/Message.h

```diff
@@ -113,6 +126,8 @@ class SOFA_HELPER_API Message
 
     /// possible levels of messages (ordered)
     enum Type {Info=0, Advice, Deprecated, Warning, Error, Fatal, TEmpty, TypeCount};
+    typedef std::set<Type> TypeSet;
```

Comment:
Ok I see, is there a way to fill "Message::AnyTypes" automatically from "enum Type", it seems really easy to have incoherences.